integration/mock-client/blockchain/bsn-irita.go

```diff
@@ -0,0 +1,64 @@
+package blockchain
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func handleBSNIritaRequest(msg JsonrpcMessage) ([]JsonrpcMessage, error) {
+	switch msg.Method {
+	case "status", "block_results":
+		rsp, ok := GetCannedResponse("birita", msg)
+		if !ok {
+			return nil, fmt.Errorf("failed to handle BSN-IRITA request for method %s", msg.Method)
+		}
+
+		return rsp, nil
```

Comment:
For now I remain it for the customized route which is needed for the mock client because the tendermint jsonrpc client seems not to support path in url.

Reply:
I see, makes sense